store/store.go

```diff
@@ -601,6 +601,19 @@ func maybeLogRetryAttempt(reqOptions *requestOptions, attempt *retry.Attempt, st
 
 }
 
+func maybeLogRetrySummary(reqOptions *requestOptions, attempt *retry.Attempt, resp *http.Response, err error, startTime time.Time) {
+	if osutil.GetenvBool("SNAPPY_TESTING") || attempt.Count() > 1 {
```

Comment:
Sounds good. Can this be brought more inline with the other variables we have?  SNAPD_DEBUG?